addon/index.js

```diff
@@ -184,9 +188,25 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
       if (get(this, 'isValid') && get(this, 'isDirty')) {
         let content = get(this, CONTENT);
         let changes = get(this, CHANGES);
+
```

Comment:
this was to be able to use "setProperties" due to the call of "setProperties" propey triggers an objects internal "dirty" state.. But, if there is no "deepSetProperties", so i have to remember all the properties that weren't set, and reset them after.